### ISSUE TITLE
Error: THREE.PropertyBinding: Trying to update property for track: .nodes.PlaneText.userData.components.0.activeSelf but it wasn't found. 

### ISSUE THREAD

**Issue by ROBYER1** [needle-tools/needle-engine-support#73]
Using Unity 2022.1.16f1, Exporter 2.25.2-pre

When using the Spatial Triggers to enable/disable animated objects, everything works fine but in the console I get these errors:

```
THREE.PropertyBinding: Trying to update property for track: .nodes.PlaneText.userData.components.0.activeSelf but it wasn't found. 
```

I can send a repro proj for this

**Comment by hybridherbst** [comment]
Don't think we need a repro for that, thanks for reporting!

**Comment by marwie** [comment]
Hey is this still an issue? Looking at https://github.com/needle-tools/needle-engine-support/issues/74 it seems like it works here ? Or is the problem here that ``activeSelf`` is actually not a valid field on a component?

**Comment by ROBYER1** [comment]
> Hey is this still an issue? Looking at #74 it seems like it works here ? Or is the problem here that `activeSelf` is actually not a valid field on a component?

Still happening in Exporter 2.27.2-pre

**Comment by marwie** [comment]
Fixed in 2.30.1-pre
